changelogs/fragments/8519.yml

```diff
@@ -0,0 +1,2 @@
+fix:
+- UI of assets table ([#8519](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8519))
```

Comment:
Can you make the change log a bit more detailed?

Reply:
Actually, it is a id->name map. In src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx, to display the workspace columns, we will need to search the name with id.  please see the below:
const getWorkspaceNameById = (workspaceId: string) => {
            for (const [name, id] of workspaceIdNameMap) {
              if (id === workspaceId) {
                return name;
              }
            }
            return workspaceId;
          };

Reply:
thanks for the comments, all updated